packages/core/src/common/quick-open-model.ts

```diff
@@ -51,7 +51,7 @@ export interface QuickOpenGroupItemOptions extends QuickOpenItemOptions {
 export class QuickOpenItem<T extends QuickOpenItemOptions = QuickOpenItemOptions> {
 
     constructor(
-        protected options: T = {} as T
+        readonly options: T = {} as T
```

Comment:
i am not sure that exposing this is a good idea. Someone can subclass `QuickOpenItem` and override methods to provide other values. We should rather call methods to collect computed values.

Reply:
> i am not sure that exposing this is a good idea.

Why? Please explain.

> Someone can subclass `QuickOpenItem` and override methods to provide other values.

Yes, someone can do that, but I needed the original `options` do be able to create a "copy". 

> We should rather call methods to collect computed values.

I do not need computed values here.